celerdata/datadog_checks/celerdata/celerdata.py

```diff
@@ -0,0 +1,17 @@
+from datadog_checks.base import OpenMetricsBaseCheckV2
+
+
+class CelerdataCheck(OpenMetricsBaseCheckV2):
+    __NAMESPACE__ = 'celerdata'
+
+    def __init__(self, name, init_config, instances):
+        super(CelerdataCheck, self).__init__(name, init_config, instances)
+
+    def get_default_config(self):
+        """
+        Returns the default OpenMetrics configuration.
+        """
+        return {
+            'metrics': ['.*'],
```

Comment:
```suggestion
            'metrics': ['.*'],
```
The best practice here is to create a dictionary mapping of the metrics we want to collect called `metrics.py`. Example: https://github.com/DataDog/integrations-core/blob/master/gitlab/datadog_checks/gitlab/metrics.py

Using the accept everything wildcard approach in my opinion a bit too vague when it comes to future troubleshooting and maintenance.

Reply:
done